openquake/calculators/scenario_risk.py

```diff
@@ -82,20 +82,6 @@ def scenario_risk(riskinputs, riskmodel, rlzs_assoc, monitor):
     return result
 
 
-def check_time_event(dstore):
-    """
```

Comment:
I see this has been moved, so I guess we already have a test for it. If we haven't, maybe we should add it.